configure

```diff
@@ -614,7 +616,8 @@ def host_arch_cc():
   for i in matchup:
     if i in k and k[i] != '0':
       rtn = matchup[i]
-      break
+      if (rtn != 's390'):
```

Comment:
I mean you can write it as `if rtn != 's390':` - the parentheses aren't necessary.